pages/parent/signup.tsx

```diff
@@ -174,32 +181,6 @@ export default function ParticipantInfo({
         [],
     );
 
-    const medicationDetails = isOnMedication ? (
-        <Box mt={4}>
-            <FormControl id="details">
-                <FormLabel>Please provide any details if necessary</FormLabel>
-                <Input
-                    placeholder="Details"
-                    onChange={(e) => setMedication(e.target.value)}
-                    value={medication}
-                />
-            </FormControl>
-        </Box>
-    ) : null;
-
-    const allergyDetails = hasAllergies ? (
-        <Box mt={4}>
-            <FormControl id="details">
-                <FormLabel>Please provide any details if necessary</FormLabel>
-                <Input
-                    placeholder="Details"
-                    onChange={(e) => setAllergies(e.target.value)}
-                    value={allergies}
-                />
-            </FormControl>
-        </Box>
-    ) : null;
-
```

Comment:
I think we actually want to keep these, it corresponds to the following form which has been removed, it seems like a regression from https://github.com/uwblueprint/social-diversity-for-children/pull/80. If you could, please ensure that the parent/signup flow matches exactly of that in our Figma.

![image](https://user-images.githubusercontent.com/44826218/135855791-3500b525-9d01-40b3-b1b1-d7ded07334f7.png)

But we have this currently for participant health form, which doesn't make sense:
![image](https://user-images.githubusercontent.com/44826218/135856958-0bf24278-9760-4575-9414-e34e02d2cb42.png)


And also do the required validations here.

Reply:
Yes it was likely something from #80